packages/features/bookings/Booker/utils/layout.ts

```diff
@@ -4,3 +4,19 @@ import { bookerLayoutOptions } from "@calcom/prisma/zod-utils";
 export const validateLayout = (layout?: BookerLayouts | null) => {
   return bookerLayoutOptions.find((validLayout) => validLayout === layout);
 };
+
+export const userNamesInDescription = (users) => {
+  const numberOfUsers = users.length;
+  if (numberOfUsers === 1) {
+    return `${users[0].name}`;
+  } else if (numberOfUsers === 2) {
+    return `${users[0].name} and ${users[1].name}`;
+  } else {
+    let listOfUsers = "";
+    for (let i = 0; i < numberOfUsers - 1; i++) {
```

Comment:
Please check the ticket description, it's mentioned
"if its more, it's "Firstname 1, Firstname2 & more"